packages/core/constants/index.ts

```diff
@@ -11,6 +11,12 @@ export enum FileExplorerServiceBaseUrl {
     PRODUCTION = "https://production.int.allencell.org",
 }
 
+export enum AicsLoadBalancerBaseUrl {
+    LOCALHOST = "http://localhost:8080",
+    STAGING = "http://stg-aics.corp.alleninstitute.org",
+    PRODUCTION = "http://aics.corp.alleninstitute.org",
+}
```

Comment:
May or may not work for this case, but for the MMS endpoint, we're intending to use a mapping of the current FES url to an MMS url instead of having to intialize the app with an additional URL
https://github.com/AllenInstitute/biofile-finder/blob/8a0f1194a6f0b438c4c4838b1efa72c8dcac8cc0/packages/core/services/FileService/HttpFileService/index.ts#L36-L40

Reply:
Resolved by https://github.com/AllenInstitute/biofile-finder/pull/339